github4s/shared/src/main/scala/github4s/app.scala

```diff
@@ -29,5 +29,6 @@ object app {
   type COGH02[A]   = Coproduct[GistOp, COGH01, A]
   type COGH03[A]   = Coproduct[IssueOp, COGH02, A]
   type COGH04[A]   = Coproduct[AuthOp, COGH03, A]
-  type GitHub4s[A] = Coproduct[GitDataOp, COGH04, A]
+  type COGH05[A]   = Coproduct[GitDataOp, COGH04, A]
+  type GitHub4s[A] = Coproduct[PullRequestOp, COGH05, A]
```

Comment:
We could look into using the [trick they use in quasar](https://github.com/quasar-analytics/quasar/blob/master/foundation/src/main/scala/quasar/fp/coproduct.scala) to build nested coproducts.

That way we could write : 
```scala
type Github4s[A] = (
        PullRequestOp
    :\: GitDataOp
    :\: AuthOp
    :\: IssueOp
    :\: GistOp
    :\: RepositoryOp
    :/: UserOp
)#M[A]
```

But if we are moving to freestyle once it is released, it is probably not worth it.